mod_reforged/hooks/items/shields/shield.nut

```diff
@@ -20,7 +20,7 @@
 			id = 8,
 			type = "text",
 			icon = "ui/icons/fatigue.png",
-			text = ::Reforged.Mod.Tooltips.parseString("The defense values drop linearly as you build [Fatigue|Concept.Fatigue] up to " + ::MSU.Text.colorRed((maxReduction * 100) + "%") + " at maximum [Fatigue|Concept.Fatigue]")
+			text = ::Reforged.Mod.Tooltips.parseString("Except when using [Shieldwall|Skill+shieldwall], the defense values drop linearly as you build [Fatigue|Concept.Fatigue] up to " + ::MSU.Text.colorRed((maxReduction * 100) + "%") + " at maximum [Fatigue|Concept.Fatigue]")
```

Comment:
`::MSU.Text.colorRed((maxReduction * 100) + "%")`
could be turned into
`::MSU.Text.colorizeMult(maxReduction)`